model-archiver/model_archiver/tests/integ_tests/test_integration_model_archiver.py

```diff
@@ -40,16 +40,28 @@ def delete_file_path(path):
         pass
 
 
-def run_test(test, cmd):
-    it = test.get("iterations") if test.get("iterations") is not None else 1
+def run_test(test, args, mocker):
```

Comment:
do you mind giving the rest of the team a brief presentation on test mocking and pytest more generally? I'd for one certainly really enjoy it

Reply:
Happy to do so